R/use_extendr.R

```diff
@@ -19,7 +19,12 @@ use_extendr <- function(path = ".", quiet = getOption("usethis.quiet", FALSE)) {
   pkg_name <- pkg_name(path)
 
   src_dir <- rprojroot::find_package_root_file("src", path = path)
+  r_dir <- rprojroot::find_package_root_file("R", path = path)
   wrappers_file <- rprojroot::find_package_root_file("R", "extendr-wrappers.R", path = path)
+  if (!dir.exists(r_dir)) {
+    ui_v("Writing {.file R/}")
```

Comment:
I think this needs to be guarded with `if(!isTRUE(quiet))`, similar to how its is done for `src_dir` (line 30).

Reply:
You're right, good catch!